src/org/opensha/sha/earthquake/faultSysSolution/ruptures/util/RupSetDiagnosticsPageGen.java

```diff
@@ -685,8 +685,14 @@ public void generatePage() throws IOException {
 					summary.compMeta.scalars.add(new ScalarRange(compScalars));
 					compScalarVals.add(compScalars);
 				}
-				plotRuptureHistograms(resourcesDir, "hist_"+scalar.name(), table, inputScalars,
-						inputUniques, compScalars, compUniques);
+				try {
+					plotRuptureHistograms(resourcesDir, "hist_"+scalar.name(), table, inputScalars,
+							inputUniques, compScalars, compUniques);
+				} catch( IllegalStateException wee) { 
```

Comment:
if you want to leave this in, the block should be formatted, and I would recommend printing out the exception too.

Reply:
I've been nursing this file along, and committed it accidentally. I'll fix up the exception handlers as you suggest :)